src/early_multicellular_stage/editor/CellBodyPlanEditorComponent.cs

```diff
@@ -390,6 +390,13 @@ public override void OnFinishEditing()
         editedSpecies.UpdateNameIfValid(newName);
 
         behaviourEditor.OnFinishEditing();
+
+        foreach (var cellType in cellTypeSelectionButtons.Values)
+        {
+            // During multicellular editing, we don't reposition cells to the origin
+            // because it messes up the aciton history, so do it here once editing is finished
+            cellType.CellType.RepositionToOrigin();
```

Comment:
This is not fully correct because rotation speed calculation (and maybe some other stats are dependent on organelle positions). So this re-apply of the edit would actually need to basically do everything that CellEditorComponent does on finishing editing.

Reply:
I double-checked that rotation speed is the only thing in CellEditorComponent.OnFinishEditing that depends on the organelle positions, so I just started updating that in CellBodyPlanComponent.OnFinishEditing too.